website/www/site/content/en/blog/2016-02-22-beam-has-a-logo.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Apache Beam has a logo!"
+slug: "beam-has-a-logo.html"
+date:   2016-02-22 10:21:48 -0800
+excerpt_separator: <!--more-->
+categories: beam update website
```

Comment:
I see that apart from `categories: blog`, each blog will have a different URL corresponding to its categories. I will point out some of them below. I think you should ask Aizhamal if she is ok with it, although I'm not sure whether it breaks other places if we change the URLs. 😄

Reply:
Oh no 😱 I didnt notice

Reply:
No, for sure we can't change urls